boilerplate/babel.config.js

```diff
@@ -1,7 +1,6 @@
 /** @type {import('@babel/core').TransformOptions['plugins']} */
-const plugins = [
-  /** react-native-reanimated web support @see https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/installation/#web */
-  "@babel/plugin-proposal-export-namespace-from",
+const plugins = [  
+  "@babel/plugin-transform-export-namespace-from",
```

Comment:
I'd like to leave the comment in here for details on why this is here - you can update the url, though:

https://docs.swmansion.com/react-native-reanimated/docs/guides/web-support/